.vscode/extensions.json

```diff
@@ -2,6 +2,17 @@
   "recommendations": [
     "editorconfig.editorconfig",
     "redhat.vscode-yaml",
-    "biomejs.biome"
+    "biomejs.biome",
+    "github.remotehub",
+    "ms-vscode.remote-repositories",
+    "github.vscode-pull-request-github",
+    "gitlab.gitlab-workflow",
+    "eamodio.gitlens",
+    "mhutchie.git-graph",
+    "streetsidesoftware.code-spell-checker",
+    "bradlc.vscode-tailwindcss",
+    "natqe.reload",
+    "johnpapa.vscode-peacock",
```

Comment:
This seems unnecessary as a recommendation. Not problem with people installing for themselves though

Reply:
I add it as a suggestion for those who might not know it exists and could find it helpful.

Reply:
Same here too, I'd make this an informal recommendation at something like a dev sync or you could even create a Millipedia page that highlights certain tools that provide a better DX, instead of prescribing them through VSCode

Reply:
And, this is a recommendation list, not a requirement. So it's kind of a toss up here -- we can take it or leave it. I have started a [page in Millipedia](https://outline.accelint.dev/doc/environment-setup-iCcmkcSuNa) for this type of thing.